### ISSUE TITLE
@start/cli not working on windows

### ISSUE THREAD

**Issue by roddolf** [nextools/metarepo#29]
Installing `@start/cli` and running with `yarn start` throws:
```
%PROJECT%\node_modules\.bin\start
The system cannot find the path specified.
```

If I try to run the bin directly it says:
```
Line |
  16 |    & "/bin/sh$exe"  "$basedir/../@start/cli/node/index.js" $args
     |      ~~~~~~~~~~~~~
     | The term '/bin/sh' is not recognized as the name of a cmdlet, function, script file, or operable
     | program. Check the spelling of the name, or if a path was included, verify that the path is correct
     | and try again.
```

Also, if using globally or with npx, will not work since Windows has a built-in command for `start`:
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/start

**Comment by deepsweet** [comment]
Thanks. I see.

>The term '/bin/sh' is not recognized as the name of a cmdlet

That's because we use quite a [tricky approach](https://github.com/nextools/metarepo/blob/63d0d709d4600d5b7551b29833d346117587d94a/packages/start/cli/src/index.js#L1-L3) to pre-include Node.js v10 `--experimental-worker` flag automatically without asking user to do that.

Would it work for you locally (especially with Node.js v10) if you:

1. change first 2 lines in `./node_modules/@start/cli/build/node/index.js` to just `#!/usr/bin/env node`
2. wrap `start` cli into `start` script in in your root `package.json` like this:

```json
"scripts": {
  "start": "NODE_OPTIONS=--experimental-worker ./node_modules/@start/cli/build/node/index.js"
}
```
3. `yarn start ...`

If it works then I think it'd be an "official" approach from our side until we drop Node.js v10 once v14 become an LTS.

>since Windows has a built-in command for `start`

That's why while ago I've renamed it to `start-runner` and then for some reason back to `start` when fully refactored. Not sure what's the best approach here, maybe just having 2 bin aliases `start` and `start-runner` is the way to solve "if using globally or with npx" problem.

**Comment by deepsweet** [comment]
Current plan is to:

* drop Node.js v10 much sooner than v14 become an LTS in October (solves `"/bin/sh$exe"` issue)
* publish Start CLI with 2 `start` and `start-runner` bin aliases so that Windows users could use the later ("solves" the built-in `start` command issue)

**Comment by roddolf** [comment]
I do check the approach of changing to use the start script and worked. But since I did have to change that every time I reinstalled the modules I better change the shell executor to use bash from WSL. But would be nice to be supported without that.

I also like the two aliases, since I use the npm scripts to also run certain `start` tasks.

**Comment by roddolf** [comment]
@deepsweet 
I just saw that Node.js v10 support has been dropped, so I updated but now this error shows when trying to run:
![image](https://user-images.githubusercontent.com/14943928/102183668-0a465d00-3e63-11eb-8bf4-6ac795d5fb50.png)

It looks like the script is trying to be run directly without Node.
The PowerShell script generated on `.bin` folder:
```
#!/usr/bin/env pwsh
$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent

$exe=""
if ($PSVersionTable.PSVersion -lt "6.0" -or $IsWindows) {
  # Fix case when both the Windows and Linux builds of Node
  # are installed in the same directory
  $exe=".exe"
}
& "$basedir/../@start/cli/node/index.js"   $args
exit $LASTEXITCODE
``

**Comment by deepsweet** [comment]
Hi!

Should be fixed now with `@start/cli@2.0.1`

**Comment by roddolf** [comment]
I can validate that it's now fixed on v2.0.1

Thank you for the fast response.

**Comment by deepsweet** [comment]
Glad it helped! Feel free to re-open this issue.
